src/main/java/picard/sam/SetNmMdAndUqTags.java

```diff
@@ -73,6 +73,9 @@
     @Option(doc = "Whether the file contains bisulfite sequence (used when calculating the NM tag).")
     public boolean IS_BISULFITE_SEQUENCE = false;
 
+    @Option(doc = "Only set the UQ tag, ignore MD and NM")
+    public boolean SET_ONLY_UQ = false;
```

Comment:
I don't have a use-case for only UQ or other individual tags, but my thought was coming from hearing @vdauwera's philosophy to have a multi-purpose tool.  If it's easy to add in, then go for it, otherwise, use the boolean.  My suggestions are not gospel.